### PR TITLE
Dropout paper draft

### PR DESCRIPTION
**Pull request by jasonm**
Review of ["Dropout: A Simple Way to Prevent Neural Networks from Overfitting", Srivastava et al 2014](http://jmlr.org/papers/volume15/srivastava14a/srivastava14a.pdf)

Paper TLDR:
* Dropout is a regularization technique. When it works, it improves generalization by reducing overfitting.
* Dropout works by blanking out subsets of the network at training time to produce many thinned variants of the network (all networks share weights). Perform test/prediction either (1) from the unthinned network by scaling weights or (2) by sampling any/all networks and averaging.
* Dropout works well on dense/feedforward layers.
* Dropout works well across many task types (vision, speech recognition, document classification, computational biology)
* Dropout does not work well on CNN/RNN layers without additonal treatment. (Discussed in other later papers which I reference)
* Dropout generally lets you increase learning rate and momentum.
* Dropout increases feature sparsity.

I would like to learn more about:
* Bayesian neural networks
* Gaussian Processes
* Variational inference
* Using dropout to produce model uncertainty
* "Active learning" by which models direct you where to label more training data
* Restricted Boltzmann Machines and "reconstruction"

### CODE REVIEW

File: 2017-09-13-dropout-paper-review.md [hunk 1]
```diff
@@ -0,0 +1,451 @@
...
+What are the authors going to do to answer the specific question(s)?
+
+* Try a bunch of models on benchmark datasets with and without dropout.
+* Measure weight activations and produce histograms to measure the effect on
+  sparsity.
+* Compare the weight scaling approximation technique to the Monte Carlo
+  sampling technique for prediction to see how much accuracy is traded at what
+  performance cost.
+
+### Methods: Sparisty
```
Comment (JasonBenn) [inline 2017-09-13-dropout-paper-review.md]: Stray title
